src/sqsd.groovy

```diff
@@ -119,5 +119,5 @@ def handleMessage(String httpHost, String httpPath, String contentType, Message
 
     println "POST " + httpHost + httpPath + " :: " + status
 
-    status < 400
+    status == 200
```

Comment:
According to beanstalk doc:

` When the web application in the worker environment tier returns a 200 OK response to acknowledge that it has received and successfully processed the request, the daemon sends a DeleteMessage call to the SQS queue so that the message will be deleted from the queue.`